tor/rootfs/etc/s6-overlay/s6-rc.d/init-tor/run

```diff
@@ -89,6 +89,11 @@ else
     echo 'SOCKSPort 127.0.0.1:9050' >> "${torrc}"
 fi
 
+# Configure Http tunnel port
+if bashio::config.true 'http_tunnel'; then
```

Comment:
Instead of introducing another option, we can also check if the port is actually set in the add-on configuration (and default it to `null`).

Reply:
Ignore this one, you are mimicking the existing socks 👍